home/models.py

```diff
@@ -594,6 +598,12 @@ class WhatsAppTemplateCategory(models.TextChoices):
                     help_text="Each message will be sent with the text and media"
                 ),
             ),
+            (
+                "Whatsapp_Template",
+                SnippetChooserBlock(
+                    "home.WhatsAppTemplate", help_text="WhatsAppTemplate to use"
+                ),
+            ),
```

Comment:
I think we should add a test at least in `test_models` to ensure that we can save a link between the two, and then assert that it can display correctly (e.g. `wa_body`)